cidc_api/shared/gcloud_client.py

```diff
@@ -117,6 +118,111 @@ def revoke_upload_access(user_email: str):
     bucket.set_iam_policy(policy)
 
 
+def grant_download_access(user_email: str, trial_id: str, upload_type: str):
+    """
+    Give a user download access to all objects in a trial of a particular upload type.
+    """
+    member_id = f"user:{user_email}"
+    url_prefix, prefix_expression = _get_prefix_expression(trial_id, upload_type)
+    ttl_expression = _get_ttl_expression()
+
+    print(f"Granting download access on {url_prefix}* to {user_email}")
+
+    # get the current IAM policy for the data bucket
+    bucket = _get_bucket(GOOGLE_DATA_BUCKET)
+    policy = bucket.get_iam_policy(requested_policy_version=3)
+    policy.version = 3
```

Comment:
not sure why that is needed? if it is - pls add a comment in the code.

Reply:
Will do - this is just following the example in the GCS docs, so I'll link to that.